.github/workflows/main.yml

```diff
@@ -15,8 +15,10 @@ jobs:
     - uses: actions/checkout@v2
     - name: Test
       run: xcodebuild test -project Cleanse.xcodeproj -scheme Cleanse -enableCodeCoverage=YES -destination "platform=iOS Simulator,name=iPhone 8"
-    - name: Test Sample App
+    - name: Test CleanseGithubBrowser Sample App
       run: xcodebuild test -project Examples/CleanseGithubBrowser/CleanseGithubBrowser.xcodeproj -scheme CleanseGithubBrowser -destination "platform=iOS Simulator,name=iPhone 8"
+    - name: Test CleanseSwiftUI Sample App
+      run: xcodebuild test -project Examples/CleanseSwiftUIExample/CleanseSwiftUIExample.xcodeproj -scheme CleanseSwiftUIExample -destination "platform=iOS Simulator,name=iPhone 8"
```

Comment:
I think the scheme in the project file is `adoptme` instead of `CleanseSwiftUIExample`, and since there aren't any tests configured for the scheme I think just `xcodebuild build ...` will work.

Reply:
yes. it's done.